apps/game_backend/lib/game_backend/users/user.ex

```diff
@@ -64,10 +67,10 @@ defmodule GameBackend.Users.User do
       :profile_picture,
       :google_user_id
     ])
-    |> unique_constraint([:game_id, :username])
     |> cast_assoc(:unlocks)
     |> assoc_constraint(:google_user)
     |> validate_required([:game_id, :username])
+    |> validate_length(:username, min: @min_chars, max: @max_chars)
```

Comment:
More a philosophical question, but do we see any value of having `@min_chars` vs just putting `3`?  (When we are talking one single usage, multiple for sure makes more sense)

Reply:
hmmm yeah, it doesn't add anything. I'll just delete them